.changeset/eighty-pumpkins-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/ignition-core": patch
+---
+
+Add arbitrum sepolia to chain config (v-next)
```

Comment:
```suggestion
---
"@nomicfoundation/ignition-core": patch
"hardhat": patch
---

Add Arbitrum Sepolia to chain config in Ignition ([#6518](https://github.com/NomicFoundation/hardhat/pull/6518))
```

Adding the patch to hardhat lets us make the auto-release update the changelog effectively - I also like linking the issues (or PR if there is no issue).